pkgs/servers/reproxy/default.nix

```diff
@@ -11,6 +11,14 @@ buildGoModule rec {
     hash = "sha256-RB3+IU6halnk/2REh2CLDpQN7djn4Y1QuL8y8xppnQw=";
   };
 
+  patches = [
+    # Larger timeouts for tests that were failing on Hydra.
```

Comment:
```suggestion
    # Larger timeouts for tests that were failing on Hydra.
    # Included in next release.
```

Reply:
Upstream made a 0.6.0 with the patch, so I went for that. 🙂